.github/workflows/build-release-binary.yml

```diff
@@ -1,12 +1,12 @@
-name: "Build release binary"
+name: "Attach binaries to GitHub release"
```

Comment:
nit: "Build & attach"? (if it was named "Attach", I'd have some troubles finding it today)

Reply:
Feel free to send a follow-up patch :)